src/theme/themeContext.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import { ThemeProvider } from '@material-ui/core';
+import themeLight from 'src/theme/theme-light';
+import themeDark from 'src/theme/theme-dark';
+
+const Context = React.createContext(null);
+
+const ThemeConsumer = (props) => {
+
+  const switchTheme = (isDarken: boolean) => {
+    let theme = isDarken ? themeDark : themeLight
+    setThemeData({ ...themeData, theme: theme, darken: isDarken })
+  }
+
+  interface themeDataInterface {
```

Comment:
Podla konvencie by mali mena interface zacinat s velkym pismenom, casto sa tiez pouziva "I" na zaciatku na zdoraznenie, ze ide o interface

Reply:
Dal som velkym. To velke I na zaciatku tu neodporucaju ale to sa popripade dohodnite Jozom, ci sa bude davat. 
https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#interface

Reply:
Nepouzivajme tu zasataralu konveciu, ktora odporucala pomentovat interface s prefixom I